zebra-consensus/src/transaction.rs

```diff
@@ -533,6 +540,26 @@ where
 
         Ok(async_checks)
     }
+
+    /// Verifies a transaction's Orchard shielded data.
```

Comment:
Please add TODO comments about the missing Orchard checks, with tickets for each check.